lib/electric_slide/call_queue.rb

```diff
@@ -276,11 +276,9 @@ def bridge_agent(agent, queued_call)
         remove_agent agent
       end
 
-      agent.call.on_joined do 
-        agent.callback :connect, self, agent.call, queued_call
-      end
-
       agent.join queued_call
+      agent.callback :connect, self, agent.call, queued_call
```

Comment:
Ah, I see the way it's used there. @bklang Could you confirm the intent of this callback? Is it supposed to be called on agent selection, or when the agent has been joined to the caller? The readme is somewhat ambiguous.